cypress-tests/cypress/e2e/admin/pageBuilder/menus/menuCrud.cy.ts

```diff
@@ -0,0 +1,59 @@
+context("Page Builder - Menu CRUD", () => {
+    const menuName = "Test menu";
+    const menuSlug = "test-slug";
+
+    const menuNameEdit = "Testing Menu123";
+    const menuDescEdit = "This is an edited description.";
+
+    beforeEach(() => {
+        cy.login();
+        cy.pbDeleteAllMenus();
+    });
+
+    it("should be able to create, edit, and immediately delete a menu", () => {
+        cy.visit("/page-builder/menus");
+
+        // Create a menu.
+        cy.findByTestId("data-list-new-record-button").should("exist");
+        cy.findByTestId("data-list-new-record-button").click();
+
+        cy.findByTestId("pb.menu.create.name").type(menuName);
+        cy.findByTestId("pb.menu.save.button").click();
+        cy.findByText("Value is required.").should("exist");
+        cy.findByTestId("pb.menu.create.slug").type(menuSlug);
+        cy.findByTestId("pb.menu.create.description").type("This is a cool test.");
+        cy.findByTestId("pb.menu.save.button").click();
+
+        cy.findByText("Menu saved successfully.").should("exist");
+        cy.wait(500);
+
+        // Edit the menu.
+        cy.findByTestId("pb.menu.create.name").clear().type(menuNameEdit);
+        cy.findByTestId("pb.menu.create.slug").should("be.disabled");
+        cy.findByTestId("pb.menu.create.description").clear().type(menuDescEdit);
+        cy.findByTestId("pb.menu.save.button").click();
+        cy.findByText("Menu saved successfully.").should("exist");
+        cy.wait(500);
+
+        // Assert the menu is being displayed properly on the right side of the screen.
+        cy.findByTestId("default-data-list").within(() => {
+            cy.get("li")
+                .first()
+                .within(() => {
+                    cy.findByText(menuNameEdit).should("exist");
+                    cy.findByText(menuDescEdit).should("exist");
+                    cy.get("button").click({ force: true });
```

Comment:
Let's use a test ID here. 

The moment you're forced to use a vague selector like this one, just ping me and I'll add a test ID.

Reply:
Will add a test ID now here... hold on...

Reply:
![image](https://github.com/webiny/webiny-js/assets/5121148/c82e58b3-4b80-48b4-8097-2d62b4ccd0f2)

```
pb-menus-list-delete-menu-btn
```